client/my-sites/checkout/src/components/item-variation-picker/item-variation-dropdown.tsx

```diff
@@ -216,7 +216,12 @@ function ItemVariantOptionList( {
 		? variants[ 0 ]
 		: variants.find( ( variant ) => variant.productId === selectedItem.product_id );
 	return (
-		<OptionList role="listbox" tabIndex={ -1 } detached={ isStreamlinedPrice }>
+		<OptionList
+			id="item-variant-listbox"
```

Comment:
If there are multiple items that have a `ItemVariationPicker` on the page (this doesn't happen often but it does happen), then this `id` might clash. Maybe we can give it an id tied to the product to make it unique?

Reply:
Yep, this is exactly the potential issue which I mentioned here: https://github.com/Automattic/wp-calypso/pull/104133#pullrequestreview-2918491139

I think it's good idea, recommended by Copilot - to use random id or try to avoid it at all.

But, as I see - we render the list with `id` only for opened dropdown, so the conflict is not possible, since only one dropdown is opened at once, but still - it's not nice to use ids, moreover using hardcoded ids.

To reproduce it - try to add domain:<br />
<img width="672" alt="Screenshot 2025-06-11 at 22 07 26" src="https://github.com/user-attachments/assets/8b3c4980-2807-4145-be32-ae005f30c38a" />

Reply:
You could do something like:

```suggestion
			id={ 'item-variant-listbox-' + selectedItem.uuid }
```

Reply:
Thanks for the suggestion and reviews, I have applied the changes in https://github.com/Automattic/wp-calypso/pull/104133/commits/8a3e60301262c97b6b39004f80ecfb9e51b87df7